fileworker.py

```diff
@@ -0,0 +1,39 @@
+from handlers import TxtHandler, JsonHandler
+
+
+class FileWorker:
+    def __init__(self, file_path):
+        self.file_path = file_path
+        self.file_types = {'json': JsonHandler, 'txt': TxtHandler}
+
+    def file_extension(self):
+        if self.file_path.endswith('json'):
+            return self.file_types.get('json')
+        elif self.file_path.endswith('txt'):
+            return self.file_types.get('txt')
+        else:
+            return 'error'
+
+    def read(self):
+        reader = self.file_extension()
+        return reader(self.file_path).read()

```

Comment:
doesnt it seem strage that you create instance of handler into `read` method? 🤔
possible better to make instance of handler once and use it anywhere